### PR TITLE
Fix/tests

### PR DESCRIPTION
**Pull request by jechols**
This gets most tests working again

### CODE REVIEW

**Comment by jduss4** [comment]
How many tests are expected to be passing / failing?
I just ran it and got 43/52 passing.

File: docker/test.sh [hunk 1]
```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
-APP=core
+APP=${1:-core}
+VERBOSITY=${2:-0}
```
Comment (jduss4) [inline docker/test.sh]: I'm not familiar with this syntax, could you explain what it is doing?

Comment (jechols) [inline docker/test.sh]: Ooh, it's bash magic at its worst.  `${1:-foo}` means *give me $1 unless it's not initialized, in which case give me "foo"*.

**Comment by jechols** [comment]
@jduss4 To answer your test question, all failures *should* be related to JSON.  I have a separate ticket for that.  This PR wasn't clear enough, but this was just meant to get tests working again with django 1.8.  Something somewhere in my ENV was keeping them running for me, so I had no idea they were broken until I re-upped it.

File: docker/test.sh (hunk 1 above)
Comment (jduss4) [inline docker/test.sh]: "Give me negative 0" is how I was reading it, haha.  Thanks!

**Comment by jduss4** [comment]
I think that 8 of the tests are json related, though the final one that fails does not seem to be.
```
======================================================================
ERROR: openoni.core.tests.title_pull_tests (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: openoni.core.tests.title_pull_tests
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/opt/openoni/core/tests/title_pull_tests.py", line 10, in <module>
    from openoni.core.title_pull import (
  File "/opt/openoni/core/title_pull.py", line 14, in <module>
    WSKEY = settings.WORLDCAT_KEY
  File "/opt/openoni/ENV/local/lib/python2.7/site-packages/django/conf/__init__.py", line 49, in __getattr__
    return getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'WORLDCAT_KEY'
```

**Comment by jechols** [comment]
@jduss4 Oh right... yeah, that's another ticket.  That test hits a third-party API so I filed a ticket about what to do about it.  Check out #141

**Comment by jduss4** [comment]
Awesome.  Then I guess if we are happy with most of the tests running versus none of them, I can merge this.  Otherwise, we could wait and put more fixes in here?  Either way.

**Comment by jechols** [comment]
I'd like to address those in other tickets.  Just so we have *something* working again.

**Pull request merged**